src/main/java/uk/gov/justice/digital/zone/CuratedZoneLoad.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.zone;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.config.JobArguments;
+import uk.gov.justice.digital.domain.model.SourceReference;
+import uk.gov.justice.digital.exception.DataStorageException;
+import uk.gov.justice.digital.service.DataStorageService;
+import uk.gov.justice.digital.service.SourceReferenceService;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import static uk.gov.justice.digital.converter.dms.DMS_3_4_6.ParsedDataFields.OPERATION;
+
+@Singleton
+public class CuratedZoneLoad extends CuratedZone {
```

Comment:
revisit all Zone inheritance please. We should have one base class extending across all Zones

Reply:
Same as my previous comment at https://github.com/ministryofjustice/digital-prison-reporting-jobs/pull/103#discussion_r1239830426